skt/__init__.py

```diff
@@ -37,6 +37,11 @@ def stringify(v):
         return str(v)
 
 
+class RpcProtocolMismatch(Exception):
+    """Exception handler for Patchwork API mismatch errors."""
+    pass
+
+
```

Comment:
Hmm, did you intend to use this somewhere and forget?

Reply:
This exception was referenced in the code already, but it didn't exist. That caused a linting error.

Reply:
Riight. Then we need the same change in sktm, because it has the same code there (we'll also probably need a library shared by skt and sktm).

Could you fix the description, though? This is not an "Exception handler", just an "Exception". Also, please don't put a full stop after a single-sentence docstring.

Reply:
I think this should be in the first patch of the PR.